Loop/Views/BolusEntryView.swift

```diff
@@ -266,7 +266,7 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
 
     private var recommendedBolusString: String {
         guard let amount = viewModel.recommendedBolus?.doubleValue(for: .internationalUnit()) else {
-            return "-"
+            return "—"
```

Comment:
Made this "—" instead of "-" per the bug description.  But it needs design to approve.